apps/examples/cy4390x_demo/Make.defs

```diff
@@ -0,0 +1,22 @@
+############################################################################
+#
+# Copyright 2017, Cypress Semiconductor Corporation or a subsidiary of 
```

Comment:
copyright year needs to be changed from 2017 => 2018 and it's applicable to all files

Reply:
Understood. that will be changed.